projects/habitat_ovmm/README.md

```diff
@@ -28,6 +41,13 @@ These describe where objects are and where the robot starts:
 git submodule update --init data/datasets/ovmm
 ```
 
+Similar to the scene dataset setup, you may need to run the following commands if this didn't download the episodes:
```

Comment:
Same here: downloads it for @keke-220 but saves pointers which occupy very less space